api/src/providers/storage/arweave/ArweaveStorage.ts

```diff
@@ -0,0 +1,71 @@
+import { Storage, UploadResult } from '../Storage';
+import axios from 'axios';
```

Comment:
I think using axios is a downgrade, because fetch + node-fetch is 0% weight on Browser. axios makes our browser bundle bigger & that's where size matters.